### PR TITLE
Add migration formula to price offer

### PR DESCRIPTION
**Pull request by amitaibu**
#19

https://www.gitbook.com/book/amitaibu/the-gizra-way/edit#/edit/19-price-offer/price_offer.md

This PR covers mostly to migration part of the price offer. There will be a follow up for adding a more detailed overview about the merits of our Gizra-Way price offer.

### CODE REVIEW

**Comment by amitaibu** [comment]
Ready for review

File: price_offer.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| Fields (`title`, `body`) | 2 | 1 |
+| Another table with data in another language  | 1 | 10 |
+| Testing | 1 |  5 |
+
+**Total: 18 hours**
+
+### Pre-processing data
+
+There are cases that for example the body field may hold internal links or media
+that needs to be retained . Often times some massaging is needs. However we currently don't have a specific formula for this
```
Comment (RachelBaram) [inline price_offer.md:48]: What do you mean by "massaging"?

**Comment by RachelBaram** [comment]
Looks good. One note on a term that is not clear.

File: price_offer.md (hunk 1 above)
Comment (amitaibu) [inline price_offer.md:48]: It's a term that describes doing some manipulation on raw data before outputting. You push and squiz the data.

**Pull request merged**